src/components/WwwFrame/DisclaimersContentful.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+	<div class="row">
+		<ol class="text-left">
+			<li
+				id="disclaimers"
```

Comment:
id's should be unique so I would move this up to the `ol`. Then, if you want to generate an id for each item you can append the index to some string like so `:id="`disclaimer_${index}`"`